.gitignore

```diff
@@ -19,3 +19,5 @@ docs
 .DS_Store
 .vscode
 README.html
+/doc/
```

Comment:
just out of curiosity, what do `/doc/` and `/Meta/` do?

Reply:
I often build individual aspects of the package before running the full `devtools::check()`. These files were generated from a `devtools::build_vignettes()` run. That's when they were created. From the helpfile (`?devtools::build_vignettes`):


"Builds package vignettes using the same algorithm that ⁠R CMD build⁠ does. This means including non-Sweave vignettes, using makefiles (if present), and copying over extra files. The files are copied in the 'doc' directory and an vignette index is created in 'Meta/vignette.rds', as they would be in a built package. 'doc' and 'Meta' are added to .Rbuildignore, so will not be included in the built package. These files can be checked into version control, so they can be viewed with browseVignettes() and vignette() if the package has been loaded with load_all() without needing to re-build them locally."